DEPRECATED.rst

```diff
@@ -427,13 +427,15 @@ Numeric support
 Following the Release of 1.48, Numeric support in Biopython is discontinued.
 Please move to NumPy for Biopython 1.49 or later.
 
-Bio.Seq and the data property
------------------------------
+Bio.Seq, Bio.MutableSeq and the data property
+---------------------------------------------
 Direct use of the Seq object (and MutableSeq object) .data property is
 deprecated.  As of Release 1.49, writing to the Seq object's .data property
 triggered a warning, and this property was made read only in Release 1.53. In
-Release 1.55 final, accessing the .data property gives a DeprecationWarning.
-The Seq object's .data property was removed in Release 1.61.
+Release 1.55 final, accessing the .data property of a Seq object gives a
+DeprecationWarning. The Seq object's .data property was removed in Release 1.61.
+Starting from Release 1.78, accessing the .data property of a MutableSeq object
+similarly gives a DeprecationWarning.
```

Comment:
maybe "deprecation warning" or just "warning" since it isn't literally the ``DeprecationWarning ``.

Reply:
done